spec/go_runner_spec.vim

```diff
@@ -0,0 +1,38 @@
+source spec/support/helpers.vim
+
+describe "Go Runner"
+  before
+    cd spec/fixtures
+  end
+  
+  after
+    cd -
+  end
+
+  describe "when test#go#runner is not set"
+    after
+      call Teardown()
+    end
+    it "should use ginkgo if 'github.com/onsi/ginkgo' is found"
+      view ginkgo/normal_test.go
```

Comment:
It feels like the tests in this file should be against against a standard go test file making sure the auto-detection uses the go runner and not the ginkgo one. Otherwise looking good